src/Console/Command/TaskCommand.php

```diff
@@ -143,11 +145,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         try {
-            $function = $this->function->getName();
+            $function = $this->function->getClosure();
```

Comment:
Original message from @lyrixx 
> Why did you make this change? (from: $this->function->getName(), to: $this->function->getClosure())

This makes it possible to push an anonymous method as a task from an Event, without affecting functionality

Reply:
okay, thanks